cogs/assistance-cmds/regionchange.wiiu.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Region Change
+url: https://lazr1026.github.io/regionchange/#/
+author.name: Lazr1026
+thumbnail-url: https://gbatemp.net/data/avatars/o/541/541964.jpg?1674102306
```

Comment:
gotta say this image is tiny af lol
github pfp?

Reply:
Sorry just seeing this, and I said the same thing, it's from GBAtemp
 I don't know how to make it bigger lol

Reply:
We can do that, just didn't want it to be blank